src/Tizen.NUI.Components/Controls/Progress.cs

```diff
@@ -472,6 +508,20 @@ private void UpdateValue()
             }
         }
 
+        /// <summary>
+        /// Update Animation for Indeterminate mode.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        private void UpdateIndeterminateAnimation()
```

Comment:
private method can't be opened. I'm not sure if we need separate Animation method. It might be UX dependent. right?

Reply:
Yes, private method can't be used by app developers.
Currently I set the basic animation for indeterminate mode, but I'll think about adding a new public method so that app developer can change other Animation. Then, I'll create another PR separately.